### PR TITLE
Change method name 'generate' to 'createJpeg', 'increase' to 'includeBitmap', 'ensure' to 'loadLibrary', 'build' to 'getImageRequest', 'get' to 'createContext'

### PR DESCRIPTION
**Pull request by pdhung3012**
The class EmptyJpegGenerator is used to generate an empty JPEG image.  This method named "generate" is to create a new JPEG image. Thus, the method name "createJpeg" is more intuitive than "generate".

The class BitmapCounter is used to count the Bitmaps.  This method named "increase" is to check the bitmap. Thus, the method name "includeBitmap" is more intuitive than "increase".

The class BitmapCounter is used to represent GifImage.  This method named 'ensure' is to load library for image processing. Thus, the method name 'loadLibrary' is more intuitive than 'ensure'.

The class ImageUrlsRequestBuilder is used to build ImageUrlsRequest.  This method named 'build' is to get a new Image url request. Thus, the method name 'getImageRequest' is more intuitive than 'build'.

The class DefaultFrescoContext  is used to represent DefaultFrescoContext.  This method named 'get' is to create a context. Thus, the method name 'createContext' is more precise than 'get'.

### CODE REVIEW

File: vito/provider/src/main/java/com/facebook/fresco/vito/provider/DefaultFrescoContext.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
   private static @Nullable FrescoContext sInstance;
   private static @Nullable Supplier<Boolean> sDebugOverlayEnabledSupplier;
 
-  public static synchronized FrescoContext get(Resources resources) {
+  public static synchronized FrescoContext createContext(Resources resources) {
```
Comment (oprisnik) [inline vito/provider/src/main/java/com/facebook/fresco/vito/provider/DefaultFrescoContext.java]: This change doesn't make sense. It doesn't always create a new instance.

File: samples/comparison/src/main/java/com/facebook/samples/comparison/urlsfetcher/ImageUrlsRequestBuilder.java [hunk 2]
```diff
@@ -39,7 +39,7 @@ public ImageUrlsRequestBuilder addImageFormat(ImageFormat imageFormat, ImageSize
     return this;
   }
 
-  public ImageUrlsRequest build() {
+  public ImageUrlsRequest getImageRequest() {
```
Comment (oprisnik) [inline samples/comparison/src/main/java/com/facebook/samples/comparison/urlsfetcher/ImageUrlsRequestBuilder.java]: This doesn't make sense and is a huge breaking API change.

File: imagepipeline/src/main/java/com/facebook/imagepipeline/memory/BitmapCounter.java [hunk 3]
```diff
@@ -59,7 +54,7 @@ public void release(Bitmap value) {
    * @param bitmap to include in the count
    * @return true if and only if bitmap is successfully included in the count
    */
-  public synchronized boolean increase(Bitmap bitmap) {
+  public synchronized boolean includeBitmap(Bitmap bitmap) {
```
Comment (oprisnik) [inline imagepipeline/src/main/java/com/facebook/imagepipeline/memory/BitmapCounter.java]: The old name is clearer IMO.

Comment (pdhung3012) [reply imagepipeline/src/main/java/com/facebook/imagepipeline/memory/BitmapCounter.java]: Thank you for your notification. Do you think that, the other case 'generate' to 'createJpeg' improve the readability?

**Comment by defHLT** [comment]
It's unclear that the proposed names are better. Closing as it has been stale for a long time.

**Pull request closed without merging**